scenarios5/23_Akulas_Place.cfg

```diff
@@ -230,6 +230,54 @@
         {VARIABLE last_scenario 23}
         {VARIABLE amplificators_destroyed 0}
         {MODIFY_UNIT side=7 status.petrified yes}
+        {VARIABLE_OP demon_type rand (Javelineer_steel,Dark Sorcerer_steel,Duelist_steel,Longbowman_steel,Swordsman_steel,Lieutenant_steel)}
+        {GENERIC_UNIT 2 $demon_type 23 14}
+        {GENERIC_UNIT 2 $demon_type 24 16}
+        {GENERIC_UNIT 2 $demon_type 26 14}
```

Comment:
This would be far easier with a foreach on the result of `[store_locations]` with a filter for castle terrain in some radius around Akula.

Reply:
I'll check that out, thanks.

Reply:
Um, easier?  No.  Not.  At.  All.

If the syntax for [store_locations] wasn't insane enough, the gui doesn't distinguish between a lowercase L and a capital i in the terrain type.  Makes a difference though.

BTW, seems strange that Akula's in a lit castle when she's neutral and the player has to actively work at not having a predominately lawful army (Legacy of Light is nice, but with it being just about everyone's legacy along with Prophets and Celestial Messengers being lawful/illuminating...).

Reply:
> Um, easier? No. Not. At. All.

As I know myself, there would be several hexes set wrong and some adjustments to map would break it.

> BTW, seems strange that Akula's in a lit castle when she's neutral and the player has to actively work at not having a predominately lawful army

I think I made it for the purpose of making the player's predominantly lawful army less of a burden in the final fight.